datasets/assset/dataset.json

```diff
@@ -0,0 +1,266 @@
+                    },
+                    "id": {
+                        "description": "Uniek identificerend kenmerk van het object.",
+                        "type": "integer"
+                    },
+                    "guid": {
+                        "description": "Uniek nummer van het IMBOR-object (GUID)",
+                        "type": "string"
+                    },
+                    "imgeoIdentificatie": {
+                        "description": "Overerving van de geometrie van beheerobject uit bovenliggend informatiemodel of automatisch genereren.",
+                        "type": "string"
+                    },
+                    "aantalDeklagen": {
+                        "description": "Aantal deklagen bij asfaltverharding.",
+                        "type": "number"
+                    },
+                    "aantalOnderlagen": {
+                        "description": "Aantal onderlagen bij asfaltverharding.",
+                        "type": "number"
+                    },
+                    "aantalTussenlagen": {
+                        "description": "Aantal tussenlagen bij asfaltverharding.",
+                        "type": "number"
+                    },
+                    "afmeting": {
+                        "description": "Aanduiding voor de afmeting van een object in lengte, breedte en hoogte.",
+                        "type": "string"
+                    },
+                    "bagOpenbareruimteNaam": {
+                        "description": "Aanduiding van de openbare ruimte, zoals gedefinieerd in IMGeo of in de BAG, waarbinnen het beheerobject ligt.",
+                        "type": "string"
+                    },
+                    "bagWoonplaatsNaam": {
+                        "description": "Aanduiding van de woonplaats waarbinnen het beheerobject ligt. Overname uit IMGeo-bestand met woonplaats- of stadsdeelgrenzen.",
+                        "type": "string"
+                    },
+                    "beginGarantieperiode": {
+                        "description": "Jaar waarin de garantieperiode van het opgeleverde beheerobject begint.",
+                        "type": "string",
+                        "format": "date"
+                    },
+                    "bergendVermogen": {
+                        "description": "De hoeveelheid afvalwater die in de voorziening kan worden geborgen.",
+                        "type": "string"
+                    },
+                    "bgtStatus": {
+                        "description": "Binnen de BGT verplichte statusaanduiding gekoppeld aan de levenscyclus van het het BGT-object.",
+                        "type": "string"
+                    },
+                    "breedte": {
+                        "description": "Breedte van het beheerobject.",
+                        "type": "number"
+                    },
+                    "brkGmeNaam": {
```

Comment:
Afkortingen in veldnamen zijn niet toegestaan. https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#naamgeving

Als de namen te lang worden kan je eventueel een shortname attribuut gebruiken. Deze shortname wordt dan ook gebruikt voor de database objecten: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#shortname

Reply:
DIt is nog niet resolved